.gitignore

```diff
@@ -1,3 +1,4 @@
+# Ref. https://github.com/github/gitignore/blob/master/C%2B%2B.gitignore
 # Prerequisites
 *.d
```

Comment:
Both CMake and bazel do out-of(-source)-tree builds. We don't need to ignore deps or objs here.